Library/Formula/memo.rb

```diff
@@ -0,0 +1,17 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.5.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "3e047b09e91d695f1767d1dd6d179732c07a5759"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install"
```

Comment:
You probably need to specify a prefix to `install`: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19553/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_memo/